contracts/rust/adapter/Cargo.toml

```diff
@@ -5,6 +5,10 @@ version = { workspace = true }
 authors = { workspace = true }
 edition = { workspace = true }
 
+# Contract bindings docs fail to compile
+[lib]
+doctest = false
```

Comment:
Can't we add module level instruction? Guarded at bindings module only?

Reply:
I tried 

```
#[cfg(not(doctest))]
#[allow(dead_code)]
pub(crate) mod bindings;
``` 
but that just fails compile with doctests. Since we don't have any doctests in this crate, what's wrong with disabling it?

Reply:
nothing wrong. i thought it would work. it's fine.